SmartGun/CORE/Logics/HTMLTableService.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Web;
+using Database.Model;
+using MessagesLibrary;
+
+namespace CORE.Logics
+{
+    public class HTMLTableService
```

Comment:
it is not a service please rename HTMLTableService -> HTMLTableGenerator or HTMLTableCreator